.circleci/config.yml

```diff
@@ -292,44 +369,70 @@ jobs:
           command: |
             gsutil -m rsync -r ~/ng/deploy/code gs://code-angularjs-org-338b8.appspot.com
 
+  # The `deploy-docs` job should only run when all of these conditions are true for the build:
+  # - It is for the `angular/angular.js` repository (not a fork).
+  # - It is not for a pull request.
+  # - It is for the stable branch(*).
+  #
+  # *: The stable branch is the one that has the value `latest` in `package.json > distTag`.
+  deploy-docs:
+    executor:
+      name: default-executor
+    steps:
+      - skip_on_pr_and_fork_builds
+      - custom_attach_workspace
+      - init_environment
+      - skip_unless_stable_branch
+      - run: yarn grunt prepareDeploy
```

Comment:
Is this needed since the `deploy-docs` job requires the `prepare-deployment` job?

Reply:
Yes, it is still necessary, because we are not persisting all affected files to the workspace.

Currently, the `prepareDeploy` command will affect files in the `deploy/` and `scripts/docs.angularjs.org-firebase/` directories and also create a `firebase.json` file at the root directory, but we [only persist the `deploy/` directory][1].

I guess we could persist the whole `ng/` directory and not require this, though. I'll update the PR.

[1]: https://github.com/angular/angular.js/blob/295213df9537666254626ffe3e4a6013122e4802/.circleci/config.yml#L265